build_system/src/build.rs

```diff
@@ -237,18 +244,19 @@ fn build_codegen(args: &BuildArg) -> Result<(), String> {
 
     println!("[BUILD] sysroot");
     build_sysroot(
-        &mut env,
-        args,
-        &config,
+        &env,
+        args.config_info.sysroot_panic_abort,
```

Comment:
Why not pass `args` directly like before?

Reply:
I did some back and forth changes so now we can indeed put back `args` like before.